packages/payment-detection/src/eth/near-info-retriever.ts

```diff
@@ -0,0 +1,128 @@
+import { PaymentTypes } from '@requestnetwork/types';
+import ETHInfoRetriever from '../eth/info-retriever';
+import { Client } from 'pg';
+
+/**
+ * Gets a list of transfer events for an address and payment reference
+ */
+export default class NearInfoRetriever extends ETHInfoRetriever {
```

Comment:
I'm not sure it's useful to extend ETHInfoRetrieve, what do you reuse?

Reply:
OK I'll adapt. I initially thought there would be more common logic